### PR TITLE
add create retention policy task for #5

### PR DESCRIPTION
**Pull request by dongshu2013**

### CODE REVIEW

**Comment by dplummer** [comment]
Add your change to the `CHANGELOG` under the unreleased section

**Comment by dongshu2013** [comment]
Done.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ CHANGELOG inspiration from http://keepachangelog.com/.
 
 ## Unreleased
 
+## [1.1.0] - August 1, 2016
+* Support configuration of retention policies
```
Comment (dplummer) [inline CHANGELOG.md]: This should be under the Unreleased section

**Comment by dongshu2013** [comment]
Thanks for the comments! How about now?

**Pull request merged**